test/e2e/steps/given.ts

```diff
@@ -189,13 +189,14 @@ Given(
 );
 
 Given(
-  /^widget config is updated with colors and i18n$/,
+  /^widget config is updated with colors and i18n with CSP nonce$/,
   async function() {
     const config = {
       ...interactionCodeFlowconfig,
       colors: {
         brand: '#008000'
       },
+      cspNonce: 'e2e', // nonce is set in webpack.config,
```

Comment:
does the nonce have to match `nonce-playground` set in `webpack.playground.config.js`?

Reply:
nope, the one added in playground webpack config is just for debugging convenience. e2e uses another test app, see webpack.config.js in `@okta/test.app`